### PR TITLE
Added implementation of ExecutorService and ForkJoin with tests

### PR DESCRIPTION
**Pull request by vicire**

### CODE REVIEW

**Review by AleksandrPok — APPROVED** [review]

File: src/main/java/sum/ExecutorServiceSum.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+import org.apache.commons.collections4.ListUtils;
+
+public class ExecutorServiceSum {
+    public static final int THREADS = 10;
```
Comment (RomanDubovskyi) [inline src/main/java/sum/ExecutorServiceSum.java]:
```suggestion
    private static final int THREADS = 10;
```

File: src/main/java/sum/ExecutorServiceSum.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    private final List<Integer> digits;
+
+    public ExecutorServiceSum(List<Integer> digits) {
+        this.digits = digits;
+    }
+
+    public Integer calculate() {
+        ExecutorService executorService = Executors.newFixedThreadPool(THREADS);
+        List<List<Integer>> partThread = ListUtils.partition(digits, digits.size() / THREADS);
+        List<MyCallable> myCallables = partThread.stream()
```
Comment (RomanDubovskyi) [inline src/main/java/sum/ExecutorServiceSum.java]:
```suggestion
        List<Callable> myCallables = partThread.stream()
```
it's a good practice to use abstraction instead

File: src/main/java/sum/MyCallable.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package sum;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public class MyCallable implements Callable<Integer> {
```
Comment (RomanDubovskyi) [inline src/main/java/sum/MyCallable.java]: Let's name class according to its' purpose

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]

File: src/main/java/sum/ExecutorServiceSum.java (hunk 1 above)
Comment (vicire) [reply src/main/java/sum/ExecutorServiceSum.java]: Fixed

File: src/main/java/sum/ExecutorServiceSum.java (hunk 2 above)
Comment (vicire) [reply src/main/java/sum/ExecutorServiceSum.java]: Fixed

File: src/main/java/sum/MyCallable.java (hunk 3 above)
Comment (vicire) [reply src/main/java/sum/MyCallable.java]: Fixed

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**